docs/revisions.rst

```diff
@@ -76,6 +77,8 @@ latest project files.
 
 .. note::
    It is possible to have normal and develop revisions at the same time.
+   Use the regular git commands within the cloned repositories used as 
+   develop revisions to switch branches or checkout to another commit.
```

Comment:
This seems a little too long and doenst get to the point IMO. Maybe like this instead?
> To switch branches or checkout another commit in develop revisions, first call simexpal to clone the repository. Then use regular git commands within the cloned repositories.

Reply:
How do you like the updated version?